lib/Hero/Carousel.js

```diff
@@ -1,63 +1,70 @@
-import React, { useState } from "react";
-import { AiOutlineLeft, AiOutlineRight } from "react-icons/ai";
-import Swipe from "react-easy-swipe";
+import React, { useState } from 'react';
+import leftArrow from '../../public/arrow left.svg';
+import rightArrow from '../../public/arrow right.svg';
+import { useSwipeable } from 'react-swipeable';
 
-const Carousel = ({children}) => {
+const Carousel = ({ children }) => {
   const slideLength = children.length;
   const handleNextClick = () => {
-    const index = currentIndex === slideLength - 1 ? 0 : currentIndex + 1
-    setCurrentIndex(index)
-  }
+    const index = currentIndex === slideLength - 1 ? 0 : currentIndex + 1;
+    setCurrentIndex(index);
+  };
   const handleOnPrevClick = () => {
-    const index = currentIndex === 0 ? slideLength - 1 : currentIndex - 1
-    setCurrentIndex(index)
+    const index = currentIndex === 0 ? slideLength - 1 : currentIndex - 1;
+    setCurrentIndex(index);
   };
+  const handlers = useSwipeable({
+    onSwipedLeft: handleNextClick,
+    onSwipedRight: handleOnPrevClick,
+    swipeDuration: 500,
+    preventScrollOnSwipe: true,
+    trackMouse: true,
+  });
 
-  const [currentIndex, setCurrentIndex] = useState(0)
+  const [currentIndex, setCurrentIndex] = useState(0);
   return (
     <div className="m-auto">
       <div className="w-full relative select-none">
-        <AiOutlineLeft
+        <img
```

Comment:
Let's avoid using an `img` tag for an SVG when we can render the SVG tags directly in the DOM instead. We've done it for `LeftChevron` and `RightChevron` in the `ImageGallery` component. And now that I think about it, it looks like we can re-use these same SVG paths and use a different fill color.

In this case, let's move `lib/ImageGallery/LeftChevron.js` and `lib/ImageGallery/RightChevron.js` to `lib/Icons/`, change the imports in `lib/ImageGallery/ImageGallery.js` and use `lib/Icons/*` in `lib/Hero/Carousel.js`.